distribution-packages/test.sh

```diff
@@ -5,7 +5,7 @@ IFS=$'\n\t'
 set -x
 
 sdk_version="$(dotnet --version)"
-runtime_version="$(dotnet --list-runtimes | head -1 | awk '{ print $2 }')"
+runtime_version="$(dotnet --list-runtimes | grep Microsoft.NETCore.App | awk '{ print $2 }')"
```

Comment:
`dotnet --list-runtimes` lists all the runtimes, including ASP.NET Core. This filter was relying on a certain ordering that doesn't work with .NET 9 Preview 1.